bin/template/.ipynb_checkpoints/0_main_template-checkpoint.ipynb

```diff
@@ -0,0 +1,111 @@
+{
```

Comment:
<p>Maybe keep the van Zelst reference and then have something at the top like "The course materials are based on prior tutorials designed by Iris van Zelst, Mohammed Gouiza, etc following van Zelst et al. (2022)"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geodynamics-hubzero/gem-notebooks/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>